### PR TITLE
Add casper config

### PR DESCRIPTION
**Pull request by kimmylin**
* Put casper address in ChainConfig 
* Add a TxSortInterface to be used in transaction that support prioritize casper votes over high gas price

### CODE REVIEW

File: core/types/transaction.go [hunk 1]
```diff
@@ -358,27 +356,26 @@ func (s TxByNonce) Len() int           { return len(s) }
...
 func (s TxByNonce) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
+// TxSortInterface defines the interface for sorting transactions of a block
+// while providing indexed access to the underlying data structure
+type TxSortInterface interface {
+	heap.Interface
+
+	// Gets the ith tx.
+	Get(i int) *Transaction
+	// Sets the ith tx.
```
Comment (markya0616) [inline core/types/transaction.go]: In golang, doc comments need to start with function name.
So it should be:
```
Set sets the ith tx
```

File: miner/worker.go [hunk 2]
```diff
@@ -265,7 +265,13 @@ func (self *worker) update() {
 				self.currentMu.Lock()
 				acc, _ := types.Sender(self.current.signer, ev.Tx)
 				txs := map[common.Address]types.Transactions{acc: {ev.Tx}}
-				txset := types.NewTransactionsByPriceAndNonce(self.current.signer, txs)
+				num := self.chain.CurrentBlock().Number()
```
Comment (markya0616) [inline miner/worker.go]: How about from `self.current.header.Number()`?

File: core/types/transaction.go (hunk 1 above)
Comment (kimmylin) [reply core/types/transaction.go]: done. thanks

File: miner/worker.go (hunk 2 above)
Comment (kimmylin) [reply miner/worker.go]: changed

**Pull request merged**